import/OAuth2AcPkce.qml

```diff
@@ -0,0 +1,87 @@
+import QtQml 2.0
```

Comment:
Add BSD license clause to each source file (h, cpp, qml, js [if any]). The license file is located at the root level.

Reply:
Does every file need to contain the full license text, or can it simply state that it's 3-clause BSD and refer to the LICENSE file for the full text?